lib/cn/cn_tree.c

```diff
@@ -1661,9 +1661,9 @@ cn_comp_commit(struct cn_compaction_work *w)
          */
         err = cndb_record_kvset_add(
                         w->cw_tree->cndb, tx, w->cw_tree->cnid,
-                        km.km_nodeid, &km, w->cw_kvsetidv[i], km.km_hblk.bk_blkid,
-                        w->cw_outv[i].kblks.n_blks, (uint64_t *)w->cw_outv[i].kblks.blks,
-                        w->cw_outv[i].vblks.n_blks, (uint64_t *)w->cw_outv[i].vblks.blks,
+                        km.km_nodeid, &km, w->cw_kvsetidv[i], km.km_hblk_id,
+                        w->cw_outv[i].kblks.n_blks, w->cw_outv[i].kblks.blks,
+                        w->cw_outv[i].vblks.n_blks, w->cw_outv[i].vblks.blks,
```

Comment:
I thought you were going with mbid, vbid, kbid, hbid?

Reply:
I tried really hard to not rename the world.  There is one PR in this stack the renames `struct blk_list` members.